api/api.go

```diff
@@ -8,24 +8,55 @@ import (
 	"github.com/julienschmidt/httprouter"
 )
 
-// HttpGET is a utility function for making http get requests to sia with a whitelisted user-agent
+// HttpGET is a utility function for making http get requests to sia with a
+// whitelisted user-agent. A non-2xx response does not return an error.
 func HttpGET(url string) (resp *http.Response, err error) {
 	req, err := http.NewRequest("GET", url, nil)
 	if err != nil {
 		return nil, err
 	}
-	req.Header.Add("User-Agent", "Sia-Agent")
+	req.Header.Set("User-Agent", "Sia-Agent")
```

Comment:
The header uses a `map[string][]string` internally, not a `map[string]string`. So a header can have multiple values associated with it. `Set` ensures that ours is the only value, whereas `Add` appends to the `[]string`.